gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -323,6 +325,11 @@ gst_tensor_filter_install_properties (GObjectClass * gobject_class)
       g_param_spec_string ("inputtype", "Input tensor element type",
           "Type of each element of the input tensor ?", "",
           G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
+  g_object_class_install_property (gobject_class, PROP_INPUTLAYOUT,
+      g_param_spec_string ("inputlayout", "Input Data Layout",
+          "Set channel first (NCHW) or channel last layout (NHWC) for input data. "
+          "Layout of the data can be NHWC or NCHW or none for now. ",
+          "", G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
```

Comment:
Use ENUM property. Manage default value properly.

Reply:
In the current implementation using STRING property, `inputlayout` can give a different for each input. However, with ENUM, I am not sure if this is possible.

Reply:
Going to use enum - all the tensors in `other\tensors` will be forced to have the same layout for now.